include/deal.II/matrix_free/fe_point_evaluation.h

```diff
@@ -1171,6 +1260,23 @@ class FEPointEvaluation
    */
   AlignedVector<vectorized_value_type> solution_renumbered_vectorized;
 
+  /**
+   * Temporary array to store the `solution_values` passed to the evaluate()
+   * function in a format compatible with the tensor product evaluators. For
+   * vector-valued setups, this array uses a `Tensor<1, n_components,
+   * ScalarNumber>` type to collect the unknowns for a particular basis
+   * function.
+   */
+  std::vector<scalar_value_type> solution_renumbered_face;
+
+  /**
+   * Temporary array to store a vectorized version of the `solution_values`
+   * computed during `integrate()` in a format compatible with the tensor
+   * product evaluators. For vector-valued setups, this array uses a
+   * `Tensor<1, n_components, VectorizedArrayType>` format.
+   */
+  AlignedVector<vectorized_value_type> solution_renumbered_vectorized_face;
```

Comment:
Why is this needed? Cannot you use `solution_renumbered` and `solution_renumbered_vectorized`?

Reply:
@bergbauer

Reply:
Sorry, we were at Oktoberfest yesterday. You are right, currently `dofs_per_component >= 2 * dofs_per_component_face` which works even for the linear case. Afterward, we only use pointers in the kernels and don't need the correct size. We could of course play safe and resize to `std::max(size_face, size_cell) `during setup. I have removed the additional vectors.